src/server/metadata/cmds/cmds.go

```diff
@@ -143,10 +135,22 @@ func parseEditMetadataCmdline(args []string, defaultProject string) (*metadata.E
 					Key: data,
 				},
 			}
-		case "set":
-			md, err := parseData(data)
+		case "replace":
```

Comment:
I agree that “replace” is an improvement on “set.” Maybe “replace-all” is even better, because it implies that the operation affects all the metadata?